lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -2029,7 +2029,8 @@ static void replaceUsesOfBuiltinVar(Value *V, const APInt &AccumulatedOffset,
           // the load type may not match.
           // global <3 x i64>, load <6 x i32>
           VecTy = cast<FixedVectorType>(GV->getValueType());
-          if (!Index.isZero())
+          if (!Index.isZero() || DL.getTypeSizeInBits(VecTy) !=
+                                     DL.getTypeSizeInBits(Load->getType()))
```

Comment:
Can you run clang-format on your patch? Pretty sure this isn't how clang-format would format this if statement...

Reply:
I did run clang format and also the CI said it's fine

```
 Check code style / clang-format & clang-tidy (pull_request) Successful in 1m 
```

I just ran it again and it seems right:
```
$ wget -q https://patch-diff.githubusercontent.com/raw/KhronosGroup/SPIRV-LLVM-Translator/pull/2160.patch
$ patch -p1 < 2160.patch 
patching file lib/SPIRV/SPIRVUtil.cpp
patching file test/transcoding/builtin_vars_different_type.ll
patching file lib/SPIRV/SPIRVUtil.cpp
patching file lib/SPIRV/SPIRVUtil.cpp
patching file lib/SPIRV/SPIRVUtil.cpp
$ git add lib/SPIRV/SPIRVUtil.cpp test/transcoding/builtin_vars_different_type.ll
$ git clang-format
clang-format did not modify any files
```

Reply:
That's... surprising, but if it's what clang-format is happy with...